src/main/java/uk/gov/hmcts/reform/hmc/repository/ActualHearingRepository.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.hmc.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.reform.hmc.data.ActualHearingEntity;
+import uk.gov.hmcts.reform.hmc.data.HearingResponseEntity;
+
+import java.util.Optional;
+
+@Transactional(propagation = Propagation.REQUIRES_NEW)
```

Comment:
Copied your class to my PR and thinking if we really need REQUIRES_NEW or is it copy and paste?

Reply:
copy paste issues, addressed